hydromt/data_adapter/data_adapter.py

```diff
@@ -174,6 +218,10 @@ def to_dict(self):
         for k, v in vars(self).items():
             if k in ["name", "catalog_name"]:
                 continue  # do not add these identifiers
+            if (
+                k == "driver_kwargs"
+            ):  # Prevent driver_kwargs name from ending up in the data catalog
+                k = "kwargs"
```

Comment:
Why is this needed? basically have decided that "driver_kwargs" replaces the old "kwargs" right?